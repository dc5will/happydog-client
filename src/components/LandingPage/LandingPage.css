* {
    box-sizing: border-box;
    margin: 0;
}

h2 {
    font-size: bold;
    text-transform: uppercase;
    font-size: 26px;
}

body {
    font-family: 'Raleway', sans-serif;
    font-size: 18px;
    color: #444;
    text-align: center;
    line-height: 1.5;
}

section {
    min-height: 200px;
    padding: 15px;
    background-color: rgb(224, 194, 129);
}

section:nth-child(even) {
    background-color: rgb(245, 229, 177);
}

p {
    margin: 0 auto;
    max-width: 650px;
    margin-bottom: 40px;
    line-height: 1.7;
}

p+p {
    margin-top: 15px;
    margin-bottom: 15px;
}

button {
    background-color: rgb(192, 81, 81);
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
}


ul {
    list-style-type: none;
    margin: 10px;
    padding: 0;
    font-size: 16px;
    line-height: 1.5;
}

.bannerBackground {
    background: url(../../images//happydogBanner.png) no-repeat center;
    background-size: cover;
    opacity: 0.85;
    height: 400px;
    max-width: 100%;
}



