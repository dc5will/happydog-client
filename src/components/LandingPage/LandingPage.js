import React, { Component } from "react";
import "./LandingPage.css";
import Header from "../../components/Header/Header";
import Footer from '../../components/Footer/Footer';

export default class LandingPage extends Component {
  render() {
    return (
      <>
        <Header />
        <section>
          <header>
            <h3>Is Your Dog Happy?</h3>
          </header>
          <p>
            Take the of making sure your dogs are happy by keeping
            track of all of your dogs needs and keep a record and journal for
            your beloved canine friend.
          </p>
        </section>

        <section>
          <header>
            <h3>Daily Checklist</h3>
            <p>
              Sometimes life is hectic and we're not sure if our furry family
              members have been taken care of. Utilize the easy to use daily
              checklist with your family to make sure someone in your family has
              fed and/or walked your dog.
            </p>
          </header>
        </section>

        <section>
          <header>
            <h3>Secured Notes</h3>
            <p>
              Keep important notes or memos for future vet visits or dog
              sitters. Consolidate all of your dog's important information such
              as vaccination records, allergies, special care instructions, and
              memories.
            </p>
          </header>
        </section>
        <Footer />
      </>
    );
  }
}
