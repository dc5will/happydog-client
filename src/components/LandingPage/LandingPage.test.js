import React from "react";
import { shallow } from "enzyme";
import LandingPage from "./LandingPage";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe(`LandingPage component`, () => {
  it("renders without crashing", () => {
    shallow(<LandingPage />);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <LandingPage />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
