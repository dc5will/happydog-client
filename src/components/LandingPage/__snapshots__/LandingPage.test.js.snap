// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the UI as expected 1`] = `
Array [
  <nav
    className="Header"
  >
    <h1
      className="happyDogLogo"
    >
      <a
        href="/my-notes"
        onClick={[Function]}
      >
        Happy Dog
      </a>
    </h1>
    <div
      className="Header_not-logged-in"
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
      <a
        href="/register"
        onClick={[Function]}
      >
        Register
      </a>
    </div>
  </nav>,
  <section>
    <header>
      <h3>
        Is Your Dog Happy?
      </h3>
    </header>
    <p>
      Take the guesswork out of making sure your dogs are happy by keeping track of all of your dogs needs and keep a record and journal for your beloved canine friend.
    </p>
  </section>,
  <section>
    <header>
      <h3>
        Daily Checklist
      </h3>
      <p>
        Sometimes life is hectic and we're not sure if our furry family members have been taken care of. Utilize the easy to use daily checklist with your family to make sure someone in your family has fed and/or walked your dog.
      </p>
    </header>
  </section>,
  <section>
    <header>
      <h3>
        Secured Notes and Tasks
      </h3>
      <p>
        Keep secured important notes or tasks for future vet visits or dog sitters. Consolidate all of your dog's important information such as vaccination records, allergies, special care instructions, and memories.
      </p>
    </header>
  </section>,
  <footer
    className="footer"
  >
    <div
      className="footer_brand-container"
    />
  </footer>,
]
`;
